packages/core/src/parser/StyleParser.test.ts

```diff
@@ -149,18 +149,41 @@ where IsSubset(   );
     const prog = `
 forall Set A, B; Map f
 with Set C, D; Map \`g\`
-where C := intersect ( A, B, Not(f) ) ; IsSubset( A, B ); IsSubset( Union(A,B), C); Intersect (   )
```

Comment:
Would be a good idea to add some tests for alias expression locations, e.g.:
```
forall Set A, B; Map f
with Set C, D; Map \`g\`
where C := intersect ( A, B, Not(f) ) ; IsSubset( A, B ) as SubsetAB; IsSubset( Union(A,B), C) as nested_Subset; Intersect (   )
```
...and failure cases like:

```
forall Set A, B; Map f as Const -- should fail because decls can't be aliased
```

Reply:
What do you mean by alias expression locations exactly? @wodeni